src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -142,6 +142,7 @@ public function formatAndWrap(string $message, int $width)
         $offset = 0;
         $output = '';
         $tagRegex = '[a-z][a-z0-9,_=;-]*+';
+        $length = 0;
```

Comment:
Minor comment: should we rename `$length` as `$maxLength` ? "length" is too generic, and in this case is confusing because we are also using "width" --> `$this->applyCurrentStyle(..., ..., $width, $length)`

Reply:
`$currentLineLength`? It's not a max value, e.g. for `$width = 5` and `$text = '123456'`

```
12345 // $length = 0 aka "full"
6     // $length = 1
```

So we know we can add 5-1=4 more lines before the next line break

Reply:
👍 for `$currentLineLength`